lib/Parser/Parser.cpp

```diff
@@ -158,6 +160,24 @@ Token Lexer::getNextToken(std::string &ErrStr) {
     }
   }
 
+  // Parse the knownbits pattern followed by "var"
+  std::string IsKindNameVar = StringRef(Begin-4, 3);
```

Comment:
Yes, the reason is the token is returned in previous lines of code i.e. for instruction name and width. I didn't want to modify that to avoid any complexity. That's why I am just backtracking three characters to find 'var' and then parse bit pattern further, if any.